openpype/plugins/publish/extract_review.py

```diff
@@ -996,70 +993,119 @@ def get_letterbox_filters(
             l_red, l_green, l_blue
         )
         line_color_alpha = float(l_alpha) / 255
-
-        if input_res_ratio == output_res_ratio:
-            ratio /= pixel_aspect
-        elif input_res_ratio < output_res_ratio:
-            ratio /= scale_factor_by_width
-        else:
-            ratio /= scale_factor_by_height
-
+        height_letterbox = int(output_height - (output_width * (1 / ratio)))
```

Comment:
I think you broke how pillar boxes are calculated with this change. The ratio recalculation is here due to state of letterbox or pillarbox.